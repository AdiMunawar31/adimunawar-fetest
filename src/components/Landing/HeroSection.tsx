import React from "react";
import FeatureIcons from "./FeatureIcons";
import PartnerLogos from "./PartnerLogos";

const HeroSection = () => {
  return (
    <div className='bg-white relative pt-40 pb-20 lg:pt-44 dark:bg-gray-900'>
      <div className='relative xl:container m-auto px-6 md:px-12 lg:px-6'>
        <h1 className='sm:mx-auto sm:w-10/12 md:w-2/3 font-black text-blue-900 text-4xl text-center sm:text-5xl md:text-6xl lg:w-auto lg:text-left xl:text-7xl dark:text-white'>
          Connecting Lives, <br className='lg:block hidden' />{" "}
          <span className='relative text-transparent bg-clip-text bg-gradient-to-r from-blue-600 to-cyan-500 dark:from-blue-400 dark:to-cyan-300'>
            One Contact at a Time.
          </span>
          .
        </h1>
        <div className='lg:flex'>
          <div className='relative mt-8 md:mt-16 space-y-8 sm:w-10/12 md:w-2/3 lg:ml-0 sm:mx-auto text-center lg:text-left lg:mr-auto lg:w-7/12'>
            <p className='sm:text-lg text-gray-700 dark:text-gray-300 lg:w-11/12'>
              Our innovative app redefines the way you connect with your
              contacts, transforming it into a delightful breeze. Say hello to a
              world where communication flows effortlessly, and organization
              becomes a second nature. With our intuitive and powerful tools,
              you'll experience a new level of ease in managing and maintaining
              your relationships
            </p>
            <span className='block font-semibold text-gray-500 dark:text-gray-400'>
              The best companion bot for your chat app.
            </span>
            <FeatureIcons />
            <PartnerLogos />
          </div>
          <div className='mt-12 md:-mt-12 lg:absolute -right-5 lg:w-6/12'>
            <div className='relative w-full'>
              <div
                aria-hidden='true'
                className='absolute scale-75 md:scale-110 inset-0 m-auto w-full h-full md:w-96 md:h-96 rounded-full rotate-45 bg-gradient-to-r from-sky-500 to-cyan-300 blur-3xl'
              />
              <img
                src='https://tailus.io/sources/blocks/tech-startup/preview/images/globalization-cuate.svg'
                className='relative w-full'
                alt='wath illustration'
                loading='lazy'
                width={220}
                height={180}
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  );
};

export default HeroSection;
