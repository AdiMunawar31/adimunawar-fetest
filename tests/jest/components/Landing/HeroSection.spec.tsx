import React from "react";
import renderer from "react-test-renderer";
import HeroSection from "@components/Landing/HeroSection";

it("renders HeroSection correctly", () => {
  const tree = renderer.create(<HeroSection />).toJSON();
  expect(tree).toMatchSnapshot();
});
