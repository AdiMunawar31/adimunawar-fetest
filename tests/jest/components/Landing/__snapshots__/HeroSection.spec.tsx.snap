// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders HeroSection correctly 1`] = `
<div
  className="bg-white relative pt-40 pb-20 lg:pt-44 dark:bg-gray-900"
>
  <div
    className="relative xl:container m-auto px-6 md:px-12 lg:px-6"
  >
    <h1
      className="sm:mx-auto sm:w-10/12 md:w-2/3 font-black text-blue-900 text-4xl text-center sm:text-5xl md:text-6xl lg:w-auto lg:text-left xl:text-7xl dark:text-white"
    >
      Connecting Lives, 
      <br
        className="lg:block hidden"
      />
       
      <span
        className="relative text-transparent bg-clip-text bg-gradient-to-r from-blue-600 to-cyan-500 dark:from-blue-400 dark:to-cyan-300"
      >
        One Contact at a Time.
      </span>
      .
    </h1>
    <div
      className="lg:flex"
    >
      <div
        className="relative mt-8 md:mt-16 space-y-8 sm:w-10/12 md:w-2/3 lg:ml-0 sm:mx-auto text-center lg:text-left lg:mr-auto lg:w-7/12"
      >
        <p
          className="sm:text-lg text-gray-700 dark:text-gray-300 lg:w-11/12"
        >
          Our innovative app redefines the way you connect with your contacts, transforming it into a delightful breeze. Say hello to a world where communication flows effortlessly, and organization becomes a second nature. With our intuitive and powerful tools, you'll experience a new level of ease in managing and maintaining your relationships
        </p>
        <span
          className="block font-semibold text-gray-500 dark:text-gray-400"
        >
          The best companion bot for your chat app.
        </span>
        <div
          className="grid grid-cols-3 space-x-4 md:space-x-6 md:flex md:justify-center lg:justify-start"
        >
          <a
            aria-label="add to slack"
            className="p-4 border border-gray-200 dark:bg-gray-800 dark:border-gray-700 rounded-full duration-300 hover:border-cyan-400 hover:shadow-lg hover:shadow-cyan-600/20 dark:hover:border-cyan-300/30"
            href="#"
          >
            <div
              className="flex justify-center space-x-4"
            >
              <img
                alt="slack logo"
                className="w-6 h-6"
                height={128}
                loading="lazy"
                src="https://tailus.io/sources/blocks/tech-startup/preview/images/slack.png"
                width={128}
              />
              <span
                className="hidden font-medium md:block dark:text-white"
              >
                Slack
              </span>
            </div>
          </a>
          <a
            aria-label="add to chat"
            className="p-4 border border-gray-200 dark:bg-gray-800 dark:border-gray-700 rounded-full duration-300 hover:border-green-400 hover:shadow-lg hover:shadow-lime-600/20 dark:hover:border-green-300/30"
            href="#"
          >
            <div
              className="flex justify-center space-x-4"
            >
              <img
                alt="chat logo"
                className="w-6 h-6"
                height={128}
                loading="lazy"
                src="https://tailus.io/sources/blocks/tech-startup/preview/images/chat.png"
                width={128}
              />
              <span
                className="hidden font-medium md:block dark:text-white"
              >
                Google Chat
              </span>
            </div>
          </a>
          <a
            aria-label="add to zoom"
            className="p-4 border border-gray-200 dark:bg-gray-800 dark:border-gray-700 rounded-full duration-300 hover:border-blue-400 hover:shadow-lg hover:shadow-blue-600/20 dark:hover:border-blue-300/30"
            href="#"
          >
            <div
              className="flex justify-center space-x-4"
            >
              <img
                alt="chat logo"
                className="w-6 h-6"
                height={128}
                loading="lazy"
                src="https://tailus.io/sources/blocks/tech-startup/preview/images/zoom.png"
                width={128}
              />
              <span
                className="hidden font-medium md:block dark:text-white"
              >
                Zoom
              </span>
            </div>
          </a>
        </div>
        <div
          className="text-gray-600 font-bold"
        >
          <span>
            Our Partnership :
          </span>
        </div>
        <div
          className="flex gap-6 lg:gap-12 justify-between grayscale lg:w-2/3"
        >
          <img
            alt="airbnb"
            className="h-8 sm:h-10 w-auto lg:h-12"
            src="https://tailus.io/sources/blocks/tech-startup/preview/images/clients/airbnb.svg"
          />
          <img
            alt="ge"
            className="h-8 sm:h-10 w-auto lg:h-12"
            src="https://tailus.io/sources/blocks/tech-startup/preview/images/clients/ge.svg"
          />
          <img
            alt="coty"
            className="h-8 sm:h-10 w-auto lg:h-12"
            src="https://tailus.io/sources/blocks/tech-startup/preview/images/clients/coty.svg"
          />
          <img
            alt="microsoft"
            className="h-8 sm:h-10 w-auto lg:h-12"
            src="https://tailus.io/sources/blocks/tech-startup/preview/images/clients/microsoft.svg"
          />
        </div>
      </div>
      <div
        className="mt-12 md:-mt-12 lg:absolute -right-5 lg:w-6/12"
      >
        <div
          className="relative w-full"
        >
          <div
            aria-hidden="true"
            className="absolute scale-75 md:scale-110 inset-0 m-auto w-full h-full md:w-96 md:h-96 rounded-full rotate-45 bg-gradient-to-r from-sky-500 to-cyan-300 blur-3xl"
          />
          <img
            alt="wath illustration"
            className="relative w-full"
            height={180}
            loading="lazy"
            src="https://tailus.io/sources/blocks/tech-startup/preview/images/globalization-cuate.svg"
            width={220}
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
